connector/src/main/scala/com/microsoft/kusto/spark/utils/DataTypeMapping.scala

```diff
@@ -32,7 +32,7 @@ object DataTypeMapping {
     "object" -> StringType
   )
 
-  val sparkTypeToKustoTypeMap: Map[DataType, String] = Map(
+  val SparkTypeToKustoTypeMap: Map[DataType, String] = Map(
```

Comment:
This seems to duplicate KustoTypeToSparkTypeMap
THis is out of scope for your change, but just a note that it seems we can use Guava's BiMap here to remove the duplication.

Reply:
its not 1 to 1